erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -49,9 +50,12 @@ def validate_time_logs(self):
 
 			self.total_completed_qty = flt(self.total_completed_qty, self.precision("total_completed_qty"))
 
-	def get_overlap_for(self, args, check_next_available_slot=False):
+	def get_overlap_for(self, args, doctype='', check_next_available_slot=False):
```

Comment:
```suggestion
	def get_overlap_for(self, args, check_next_available_slot=False, doctype=None):
```
New keyword arguments should be preferably added to the end to avoid breaking non-kwarg function calls.  Also `None` is a better default than an empty string.